### PR TITLE
Prelim styling

### PR DESCRIPTION
**Pull request by ChinyangaTL**

### CODE REVIEW

File: routes/entryRoutes.js [hunk 1]
```diff
@@ -2,21 +2,17 @@ const express = require('express');
...
 router.route('/').post(createEntry).get(getAllEntries);
-router
-  .route('/:id')
-  .get(getEntry)
-  .delete(deleteEntry)
-  .patch(editEntry)
-  .patch(addToFavorites);
+
+// localhost:5000/api/v1/entries/:id
+router.route('/:id').get(getEntry).delete(deleteEntry).patch(editEntry);
```
Comment (loitijoseph) [inline routes/entryRoutes.js:16]: you left out add to favourites???

**Comment by loitijoseph** [comment]
Alright, noted.

I guess that's fine.

please rename css file to something else apart from styles so that we know what the css is for because we are going to have the index styles as well.

You can rename to maybe prelim.css or something

File: routes/entryRoutes.js (hunk 1 above)
Comment (ChinyangaTL) [reply routes/entryRoutes.js:16]: No, it's handled under editEntry. Patch was used so it will edit only the attributes passed to it.

**Comment by ChinyangaTL** [comment]
> Alright, noted.
> 
> I guess that's fine.
> 
> please rename css file to something else apart from styles so that we know what the css is for because we are going to have the index styles as well.
> 
> You can rename to maybe prelim.css or something

Good call, I guess we can it global.css or utilities.css, and any styling that we see ourselves repeating can go in there.
Yeah, the other .html documents will have their independent style sheets then.

File: routes/entryRoutes.js (hunk 1 above)
Comment (loitijoseph) [reply routes/entryRoutes.js:16]: Yeah, got it. That's ok

**Comment by loitijoseph** [comment]
> > Alright, noted.
> > I guess that's fine.
> > please rename css file to something else apart from styles so that we know what the css is for because we are going to have the index styles as well.
> > You can rename to maybe prelim.css or something
> 
> Good call, I guess we can it global.css or utilities.css, and any styling that we see ourselves repeating can go in there. Yeah, the other .html documents will have their independent style sheets then.

that will be perfect. please do and request Reu to merge for us. Time is gone.

**Comment by rmoswela** [comment]
good observations guys

**Comment by rmoswela** [comment]
still naming conversion, commit messages and consistency of use of small caps and big caps is still a thorn. Please resolve on next commits

**Pull request merged**